packages/client-config/API.md

```diff
@@ -145,12 +144,12 @@ type AwsAppsyncAuthorizationType = 'AMAZON_COGNITO_USER_POOLS' | 'API_KEY' | 'AW
 type AwsRegion = string;
 
 // @public
-export type ClientConfig = clientConfigTypesV1.AWSAmplifyGen2BackendOutputs;
+export type ClientConfig = clientConfigTypesV1.AWSAmplifyBackendOutputs;
 
 // @public (undocumented)
-export enum ClientConfigFileName {
+export enum ClientConfigFileBaseName {
     // (undocumented)
-    GEN2 = "amplify-outputs",
+    DEFAULT = "amplify-outputs",
     // (undocumented)
     LEGACY = "amplifyconfiguration"

```

Comment:
```suggestion
    V0 = "amplifyconfiguration"

```

Reply:
Also, it looks like the only other package where this export is used is the integration test package. Could we keep these file name internal to the client-config package?

Reply:
Revisiting this once more, I wonder if this should be `getClientConfigFileBaseName(ClientConfigVersion)` instead? That way we can abstract away the fact that V1+ is likely always going to have the same name and V0 is the special case.

Reply:
It already accept the version. My reasoning to keep it legacy/default here https://github.com/aws-amplify/amplify-backend/pull/1143#discussion_r1523744603

I'll remove it from the exports so it's not exposed by this package.

Reply:
If this is the only place that we have the `LEGACY` and `DEFAULT` terminology, then I can live with it.